models/Community.js

```diff
@@ -3,16 +3,47 @@ module.exports = (sequelize, DataTypes) => {
         name: {
             type: DataTypes.STRING,
             allowNull: false,
+            default: '',
             validate: {
-                isAlphanumeric: true,
-                len: [1, 25]
+                notEmpty: {
+                    msg: 'The community name must be filled in'
+                },
+                isAlphanumeric: {
+                    msg: 'The community name must be alphanumeric without spaces'
+                },
+                len: {
+                    args: [1, 25],
+                    msg: 'The community name must be between 1 and 25 characters'
```

Comment:
Are these messages for us? Or for the user? Because the 'community name must be filled in' is unnecessary since the community creation form's button won't work if they don't have name and title filled out (hence they can't create a community). 
The name must be between 1 and 25 is also enforced in the form creation since I only allowed for maximum length of 25 characters.

Reply:
This is for anything that digests the API. While these attributes will be verified client side as well, if we have issues with implementation or something, it will be easier to debug with a better error.